core/domain/exp_domain_test.py

```diff
@@ -277,16 +277,10 @@ def test_validation(self):
         exploration.init_state_name = ''
         exploration.states = {}
 
-        self._assert_validation_error(
-            exploration, 'between 1 and 50 characters')
-
         exploration.title = 'Hello #'
         self._assert_validation_error(exploration, 'Invalid character #')
 
         exploration.title = 'Title'
-        self._assert_validation_error(
```

Comment:
Shouldn't an empty title and category still violate strict validation? Is that the case and is it tested in exp_domain_test?